editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -28,6 +28,8 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
 
+#ifndef _3D_DISABLED
```

Comment:
There shouldn't be so many of these `#ifndef`s littered around. Instead, there should be a `3d` subfolder, and we move 3D-only files into there.

However, moving 3D files into subfolders has to wait until after https://github.com/godotengine/godot/pull/83161 and https://github.com/godotengine/godot/pull/104696

Reply:
It's resolved